processing/src/main/java/org/apache/carbondata/processing/surrogatekeysgenerator/csvbased/CarbonCSVBasedSeqGenStep.java

```diff
@@ -952,10 +951,9 @@ private String getCarbonLocalBaseStoreLocation() {
     // In that case it will have first value empty and other values will be null
     // So If records is coming like this then we need to write this records as a bad Record.
 
-    if (null == r[0] && badRecordConvertNullDisable) {
+    if (null == r[0] && badRecordslogger.isBadRecordConvertNullDisable()) {
       badRecordslogger
-          .addBadRecordsToBuilder(r, "Column Names are coming NULL", "null",
-              badRecordsLogRedirect, badRecordsLoggerEnable);
+          .addBadRecordsToBuilder(r, "Column Names are coming NULL", "null");
```

Comment:
all caller of `addBadRecordsToBuilder`, the laster parameter always passing `"null"`, it can be removed